ProiectHangman/src/HangmanDrawing.tsx

```diff
@@ -0,0 +1,127 @@
+const HEAD = (
+    <div
+      style={{
```

Comment:
```suggestion
    <div
      key="head"
      style={{
```

Ar trebui să pui și chei, ca să nu mai dea eroare în consola de browser când se desenează spânzuratul